### PR TITLE
20 profile completed raw version!

### PR DESCRIPTION
**Pull request by Allisonc369**
A raw version of all Profile UIs is completed + Enhanced some layers of the Profile Clean Architecture + ProfileTest. No mains in the UI code, so you can't run them for now.

### CODE REVIEW

**Comment by Allisonc369** [comment]
Some changed files are related to Recipe. Just focus on the Profile part, thx all!

**Review by YadingF — APPROVED** [review]
ok

File: src/main/java/entities/Profile.java [hunk 1]
```diff
@@ -64,6 +64,15 @@ public String soutCreated() {
         return result;
     }
 
+    public String soutInfo(){
```
Comment (Hclover2003) [inline src/main/java/entities/Profile.java:67]: do you mean setInfo here?

**Review by jkahn506 — APPROVED** [review]
Looks good

**Review by Hclover2003 — APPROVED** [review]
looks good. UI implementation is great too!

**Pull request merged**